src/main/java/controller/Player.java

```diff
@@ -25,10 +25,10 @@ public class Player implements KeyListener {
 		KEY_TRANSLATOR.put(KeyEvent.VK_UP, new Up());
 		KEY_TRANSLATOR.put(KeyEvent.VK_DOWN, new Down());
 
-		KEY_TRANSLATOR_SECONDARY.put(KeyEvent.VK_1, new Left());
-		KEY_TRANSLATOR_SECONDARY.put(KeyEvent.VK_3, new Right());
-		KEY_TRANSLATOR_SECONDARY.put(KeyEvent.VK_5, new Up());
-		KEY_TRANSLATOR_SECONDARY.put(KeyEvent.VK_2, new Down());
+		KEY_TRANSLATOR_SECONDARY.put(KeyEvent.VK_NUMPAD5, new Left());
```

Comment:
I'd create some constants for these keys

Reply:
para mi está bien, ya son constantes... para mi no hace falta mas nada. Cierro el thread, cualquier cosa reabrilo @eldaroo